<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyBoard</class>
 <widget class="QMainWindow" name="MyBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyBoard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_np">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>I have no purpose :(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sg">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>430</y>
      <width>247</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>301</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridBoardLayout"/>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTRYING">
    <property name="title">
     <string>ChessGame</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menug">
    <property name="title">
     <string/>
    </property>
    <addaction name="separator"/>
    <addaction name="actionv"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menuversion_1_0">
    <property name="title">
     <string>version_1.0</string>
    </property>
   </widget>
   <addaction name="menuTRYING"/>
   <addaction name="menug"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menuversion_1_0"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionv">
   <property name="text">
    <string>v</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
